src/platforms/index.mdx

```diff
@@ -13,6 +13,17 @@ To report to Sentry you’ll need to use a language-specific SDK. The Sentry tea
 
 If your use-case is very specific, or not covered by Sentry, you'll find documentation on how to report events on your own using our [_SDK API_](https://develop.sentry.dev/sdk/overview/).
 
+<Note>
+
+Sentry SDKs are not officially supported nor recommended for use in third-party SDKs and libraries due to technical limitations:
+
+- Sentry SDKs are not designed to be used in third-party SDKs and libraries.
```

Comment:
@shanamatthews @vivianyentran would be possible to rephrase this sentence less ambiguously?

I'll explain what we want to achieve with this sentence:

We don't officially support/recommend Sentry SDKs being added to 3rd party SDKs and libraries, that is correct, for example, let's say that Stripe SDK embeds the Sentry SDK, customers add the Stripe SDK, customers also add the Sentry SDK, so technically there are 2 Sentry SDKs installed and this is a problem.

On the other hand, it's ok that 3rd party SDKs leverage Sentry SDKs, for example, https://github.com/nuxt-community/sentry-module, technically `nuxt/sentry` is also an SDK/Library, and this is fine, because in this case, `nuxt/sentry` acts as a Sentry SDK itself (but maintained by the community since we don't offer support to Nuxt ourselves), people would not add Sentry SDK directly again because this is the function of `nuxt/sentry`, and in this case, we do support and recommend.

In our sentences, we wanna address only the first use case, but not the 2nd, how can we phrase this better?

Maybe @lforst can add more context, let us know if this makes sense.

Reply:
It sounds like the problem really comes from having more than one Sentry SDK installed on an app. What do you think about reframing the entire note like this:

It is bad practice to use Sentry SDKs in third-party plugins, packages, and libraries (such as payment SDKs or embeddable widgets) meant to be consumed by other apps. If an app uses a Sentry SDK, as well as a third-party package with its own Sentry SDK installed, then the following issues can occur: 

- Version conflicts may occur from having multiple Sentry SDKs installed in the same application.
- Signal handlers may overwrite each other, leading to one of the Sentry SDKs operating incorrectly. 
- Third-party SDKs and libraries may leak users’ PII to the third-party’s Sentry organization.

Reply:
Thanks, applied the suggestion.